src/page-header/demos/enUS/basic.demo.md

```diff
@@ -69,15 +69,15 @@ export default defineComponent({
       },
       options: [
         {
-          label: 'Urge Update',
+          label: 'What?',
           key: '1'
         },
         {
-          label: 'Urge Update',
+          label: 'It can also',
           key: '2'
         },
         {
-          label: 'Urge Update',
+          label: 'Have a menu?',
           key: '3'
```

Comment:
By design. I'm not sure it is a exact presentation.

Urge update means "let them know we want them to post a new episode"